qiskit_experiments/test/mock_iq_backend.py

```diff
@@ -31,7 +33,49 @@
 )
 
 
-class MockRestlessBackend(FakeOpenPulse2Q):
+class FakeOpenPulse2QV2(FakeBackendV2):
+    """BackendV2 version of FakeOpenPulse2Q"""
+
+    def __init__(self):
+        # FakeOpenPulse2Q has its data hard-coded rather than in json files. We
+        # prepopulate the dicts so that FakeBackendV2 does not try to load them
+        # from files.
+        #
+        # We have to use a hack to populate _conf_dict
+        backend_v1 = FakeOpenPulse2Q()
+        self._conf_dict_real = backend_v1._configuration.to_dict()
+        super().__init__()
+        self._props_dict = backend_v1._properties.to_dict()
+        self._defs_dict = backend_v1._defaults.to_dict()
+        self._defaults = backend_v1._defaults
+
+        # Workaround a bug in FakeOpenPulse2Q. It defines u1 on qubit 1 in the
+        # cmd_def in the defaults json file but not in the gates in the
```

Comment:
```suggestion
        # cmd_def in the defaults dictionary but not in the gates in the
```
there is no json file for `FakeOpenPulse2Q`.